packages/types/src/payment/service.ts

```diff
@@ -1,3 +1,121 @@
 import { IModuleService } from "../modules-sdk"
+import { Context } from "../shared-context"
+import {
+  CreatePaymentCollectionDTO,
+  CreatePaymentDTO,
+  CreatePaymentSessionDTO,
+  SetPaymentSessionsDTO,
+  UpdatePaymentCollectionDTO,
+  UpdatePaymentDTO,
+} from "./mutations"
+import {
+  FilterablePaymentCollectionProps,
+  PaymentCollectionDTO,
+  PaymentDTO,
+} from "./common"
+import { FindConfig } from "../common"
 
-export interface IPaymentModuleService extends IModuleService {}
+export interface IPaymentModuleService extends IModuleService {
+  /* ********** PAYMENT COLLECTION ********** */
+
+  createPaymentCollection(
+    data: CreatePaymentCollectionDTO[],
+    sharedContext?: Context
+  ): Promise<PaymentCollectionDTO[]>
+  createPaymentCollection(
+    data: CreatePaymentCollectionDTO,
+    sharedContext?: Context
+  ): Promise<PaymentCollectionDTO>
+
+  retrievePaymentCollection(
+    paymentCollectionId: string,
+    config?: FindConfig<PaymentCollectionDTO>,
+    sharedContext?: Context
+  ): Promise<PaymentCollectionDTO>
+
+  listPaymentCollections(
+    filters?: FilterablePaymentCollectionProps,
+    config?: FindConfig<PaymentCollectionDTO>,
+    sharedContext?: Context
+  ): Promise<[PaymentCollectionDTO[], number]>
```

Comment:
todo: shouldn't return the number here correct?

Reply:
yep, fixed